src/ts/components/index.ts

```diff
@@ -0,0 +1,13 @@
+export { default as Modal } from './Modal';
+export { default as Header } from './Header';
+export { default as Main } from './Main';
+export { default as SaveVideoSection } from './SaveVideoSection';
+export { default as Video } from './Video';
+export { default as Button } from './Button';
+export { default as Navigator } from './Navigator';
+export { default as Heading } from './Heading';
+export { default as Form } from './Form';
+export { default as Input } from './Input';
+export { default as SearchVideoSection } from './SearchVideoSection';
+export { default as Skeleton } from './Skeleton';
+export { default as NotFound } from './NotFound';
```

Comment:
```suggestion
export * from './Modal';
export * from './Header';
export * from './Main';
export * from './SaveVideoSection';
export * from './Video';
export * from './Button';
export * from './Navigator';
export * from './Heading';
export * from './Form';
export * from './Input';
export * from './SearchVideoSection';
export * from './Skeleton';
export * from './NotFound';
```
export default를 사용하지 않고 단순하게 export만 해준다면, 이런식으로 사용할 수 있답니다!

Reply:
!!!! 새로 알아갑니다 감사합니다!